src/sdgio-edit.owl

```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020239> "enhanced smallholder market access"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020239>)
 
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020240> (Increased incomes and employment)
```

Comment:
I'd say this can be lowercase, it's not a full sentence.

Reply:
@pbuttigieg I have not changed this yet. I will do it in the next PR (which will mainly be about sub-IDOs).